frontend/src/modules/DistributionPlot/settings/settings.tsx

```diff
@@ -112,7 +110,12 @@ export function Settings({ initialSettings }: ModuleSettingsProps<Interfaces>) {
     return (
         <div className="flex flex-col gap-2">
             <CollapsibleGroup title="Plot type" expanded>
-                <Dropdown value={plotType as string} options={plotTypes} onChange={handlePlotTypeChanged} />
+                <Dropdown
+                    key="plot-type"
```

Comment:
This change should not be necessary as the component is not in a list - did you get an error?

Every parent element in `content` should, however, get assigned a key.